pkg/windowsdesktop/pkg/Directory.Build.props

```diff
@@ -57,11 +41,27 @@
     <FrameworkListFileClass Include="System.Security.Permissions.dll" Profile="WindowsForms;WPF" />
     <FrameworkListFileClass Include="System.Security.Principal.Windows.dll" Profile="WindowsForms;WPF" />
     <FrameworkListFileClass Include="System.Threading.AccessControl.dll" Profile="WindowsForms;WPF" />
-    <FrameworkListFileClass Include="System.Windows.Controls.Ribbon.dll" Profile="WPF" />
     <FrameworkListFileClass Include="System.Windows.Extensions.dll" Profile="WindowsForms;WPF" />
-    <FrameworkListFileClass Include="System.Windows.Forms.Design.dll" Profile="WindowsForms" />
-    <FrameworkListFileClass Include="System.Windows.Forms.Design.Editors.dll" Profile="WindowsForms" />
-    <FrameworkListFileClass Include="System.Windows.Forms.dll" Profile="WindowsForms" />
+    <FrameworkListFileClass Include="WindowsFormsIntegration.dll" />
```

Comment:
This one doesn't have a profile. Is this a WPF reference?

Reply:
Check out https://github.com/dotnet/cli/issues/10536#issuecomment-488871926 for the meaning. No profile = integration, included when both WPF and WinForms are being used. (Both profiles = common, included when either WPF or WinForms is used.)

Reply:
...We should add a link to that issue next to the items. Easy to forget how these match up and what to search for to find out. 😄

Reply:
Thank you for the link. I added it to the source.